### PR TITLE
Makefile: switch to the new OCI container image

### PR DESCRIPTION
**Pull request by vbatts**
like
https://github.com/opencontainers/distribution-spec/commit/199cf0e5f696a09557b2d7e1ab9b065dcacf5317#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52

Signed-off-by: Vincent Batts <vbatts@hashbangbash.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -41,10 +40,11 @@ DOC_FILES := \
 FIGURE_FILES := \
 	img/media-types.png
 
-EPOCH_TEST_COMMIT ?= v0.2.0
-
 TOOLS := esc gitvalidation glide glide-vc
 
+GO15VENDOREXPERIMENT=1
+export GO15VENDOREXPERIMENT
```
Comment (cyphar) [inline Makefile:46]: My guess is we don't need this anymore right? Go 1.5 was quite a while ago. :sweat_smile:

Comment (vbatts) [reply Makefile:46]: Oh right. 😅

Comment (jonboulle) [reply Makefile:46]: lgtm after this is updated :)

**Comment by vbatts** [comment]
PTAL (cc @cyphar @jonboulle )

**Comment by jonjohnsonjr** [comment]
LGTM

**Review by jonboulle — APPROVED** [review]

**Pull request merged**